app/experimental_appconfig.go

```diff
@@ -3,10 +3,8 @@
 package app
 
 import (
-	"path/filepath"
-
-	"github.com/CosmWasm/wasmd/x/wasm"
-	wasmclient "github.com/CosmWasm/wasmd/x/wasm/client"
+	// "github.com/CosmWasm/wasmd/x/wasm"
+	// wasmclient "github.com/CosmWasm/wasmd/x/wasm/client"
```

Comment:
Why are we removing wasm from experimental build?

Reply:
since wasm is not upgraded with ibc module, we're disabling it just for now